Lab6.ipynb

```diff
@@ -0,0 +1,443 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Arden Chaing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Phys 434\n",
+    "##### November 22nd, 2020\n",
+    "##### Lab 6"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import seaborn as sns\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "matplotlib.rc('font', size=22)\n",
+    "matplotlib.rc('figure', figsize=(18, 10))\n",
+    "from scipy import stats"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Set Up"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I chose the LHC data. In order to make a fake background I decided to generate 100,000 random poisson distributed numbers using scipy.stats.poisson.rvs. I made it so the Poisson distribution has $\\mu =100$. Since I am generating a backgound with a lot of simulated data I know that the distribution is truly Poisson with $\\mu =100$ so I also create the PMF using stats.poisson.pmf and multiplying it by the number of random values I generated so it has the same scale as my simulated data rather than percentages on the y axis."
```

Comment:
Set up:
- Poisson background graph looks good.
- typos in explanation: backgound --> background
- the third sentence is a bit run on/wordy so it gets hard to read. Maybe try splitting into two sentences for easier understanding
- am I missing somewhere you used stats.poisson.pmf? I only currently see stats.poisson.rvs. Otherwise explanation makes sense